### PR TITLE
Added setting for making the due tab default on dailies

### PR DESCRIPTION
**Pull request by crookedneighbor**
Added a checkbox under the editNewTasks option in settings to make the due tab the default for dailies.

### CODE REVIEW

**Comment by crookedneighbor** [comment]
In tandem with [this PR](https://github.com/HabitRPG/habitrpg-shared/pull/340)

File: public/js/directives/directives.js [hunk 1]
```diff
@@ -61,6 +61,12 @@ habitrpg
       link: function(scope, element, attrs) {
         // $scope.obj needs to come from controllers, so we can pass by ref
         scope.main = attrs.main;
+        var dailiesView;
```
Comment (crookedneighbor) [inline public/js/directives/directives.js:64]: Was not sure if this was the best place to put this, open to moving it if anyone has suggestions for a better spot.

File: src/models/user.js [hunk 2]
```diff
@@ -268,6 +268,7 @@ var UserSchema = new Schema({
     stickyHeader: {type: Boolean, 'default': true},
     disableClasses: {type: Boolean, 'default': false},
     newTaskEdit: {type: Boolean, 'default': false},
+    dailyDueDefaultView: {type: Boolean, 'default': false},
```
Comment (crookedneighbor) [inline src/models/user.js:271]: This name okay?

**Comment by crookedneighbor** [comment]
@alys and I [discussed this setting earlier](https://github.com/HabitRPG/habitrpg/pull/3574#issuecomment-59438874).

**Comment by crookedneighbor** [comment]
UUID: `75f270e8-c5db-4722-a5e6-a83f1b23f76b`

**Comment by Alys** [comment]
@crookedneighbor Tier ++

Note that the higher tiers require more effort than the lower ones, so if you make any small changes from now on, you might find that they don't grant you extra tiers (although they still will be appreciated). There's a bit of information about that at [Contributor Rewards](http://habitrpg.wikia.com/wiki/Contributor_Rewards) but ask if you have questions.

**Comment by crookedneighbor** [comment]
@Alys Understood. Mostly just happy to help. 

Should I still post my UUID after a PR gets merged, or just when asked for?

**Comment by Alys** [comment]
I don't know... It's certainly convenient when it's added. It's up to you. :)

**Pull request merged**